prow/getting_started_deploy.md

```diff
@@ -37,6 +37,10 @@ After you saved the app, click "Generate Private Key" on the bottom
 and save the private key together with the `App ID` in the top of the
 page.
 
+## Simplified manual deployment using kind or minikube
```

Comment:
Decided to export this description to a separate file to distinguish cluster cases from localhost simple deployment.